cmd/minikube/cmd/root.go

```diff
@@ -93,6 +95,11 @@ func Execute() {
 		}
 	}
 
+	if runtime.GOOS == "darwin" && detect.IsAmd64M1Emulation() {
```

Comment:
allow ppl to do this if they specify "--force" (if they do --force only warn them and don't exit)

Reply:
It's 100% segfault right now, do we still want to allow it?

Reply:
in that case no...